src/fidesops/api/v1/urn_registry.py

```diff
@@ -77,6 +77,9 @@
 USERS = "/user"
 USER_DETAIL = "/user/{user_id}"
 
+# User Permission URLs
+USER_PERMISSIONS = f"{USER_DETAIL}/permission"
```

Comment:
Nit: Could you actually show this as the whole URL `/user/{user_id}/permission` since that's how the others are stored